modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_richdocuments.adoc

```diff
@@ -14,63 +14,82 @@ xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secu
  richdocuments secure_view_option  Enable Secure View
  richdocuments watermark_text      Watermark pattern displayed in the document
  richdocuments open_in_new_tab     Open documents in a new tab
- richdocuments secure_view_open_action_default  Open documents in Secure View with watermark by default
+ richdocuments secure_view_can_print_default    Define if documents can be printed or exported
+ richdocuments secure_view_open_action_default  Enforce displaying the watermark by default
 ----
 
 == App Configuration
 
-All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes.
+All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes. Note that values have to be set in single quotes.
 
-== Set the WOPI URL
+== Define the WOPI Server URL
 
 WOPI Server URL
 
 This command sets the WOPI Server to `[IP/URL]:port` +
-`[IP/URL]:port` can be any IP/URL plus the port on which the WOPI server can be accessed like `'a.b.c.d'` or `'http:\\a.b.c.d:8098'` or `'https:\\rd.yourdomain.com'` etc. Adding the port is only necessary when not using standard ports.
+`[IP/URL]:port` can be any IP/URL plus the port on which the WOPI server can be accessed like:
+
+- `a.b.c.d`
+- `http:\\a.b.c.d:8098`
+- `https:\\rd.yourdomain.com`
```

Comment:
```suggestion
- `http://a.b.c.d:8098`
- `https://rd.yourdomain.com`
```

why are these backslashes?

Reply:
Did not recognized them, were originally present, thanks for catching!

Reply:
Geez, totally overlooked this. Rather embarrassing.